core/secrets/secret.go

```diff
@@ -195,7 +194,7 @@ func (u *URL) String() string {
 	}
 	fullPath = append(fullPath, u.Path)
 	str := strings.Join(fullPath, "/")
-	if u.Revision > 1 {
+	if u.Revision > 0 {
```

Comment:
Probably we could define this URL like `url.URL{Scheme: "secret"}`

Reply:
ok